### ISSUE TITLE
Add NodeJS support

### ISSUE THREAD

**Issue by yuraj11** [inloop/svg2android#15]
automate svg2android from CLI

**Comment by victorhaggqvist** [comment]
This would be awesome

**Comment by yuraj11** [comment]
There's support for drag & drop multiple svg at once.

**Comment by victorhaggqvist** [comment]
Yeah sure, but kind of inconvenient for several hundred icons multiple times. Having it part of a build script would be handy.

**Comment by Ashraf-Ali-aa** [comment]
:+1:

**Comment by yuraj11** [comment]
I will appreciate If someone could implement this - I do not plan implementing this feature. After research this would need a huge code rewrite.

**Comment by snowdream** [comment]
:+1:

**Comment by bhargavms** [comment]
I would love to pick this up, if you could provide some hints on how to get started since I am a Javscript newbie here :)

**Comment by serv-inc** [comment]
@bhargavms: it seems as though the main method is [`generateCode`](https://github.com/inloop/svg2android/blob/ec5b630aa703a115d889a351ebe1471caba361f0/js/main.js#L553). It seems to also need flatten.js and svg_shape_converter.js (and jQuery for the xml parsing). You would also need some way to set the command-line arguments that are offered by the web interface, f.ex.`bakeTransforms` (look for `localStorage`). @yuraj11 : thanks for the SW, it was the only one that worked for me. Please have a look at the outline and correct if anything is wrong or could be done better.

Summary:

simple:
1. read svg from command line to data structure (from stdin, file name, pick one)
2. apply generateCode, see which methods etc are needed
3. write result to output (stdout, file, pick one)

enhanced:
- options as set via web interface
- i/o options
- ? anything else ?
